src/App.js

```diff
@@ -1,24 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import HomePage from './components/HomePage';
+import DetailsPage from './components/DetailsPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Routes>
+        <Route exact path="/" element={<HomePage />} />
+        <Route path="/country/:name" element={<DetailsPage />} />
+      </Routes>
```

Comment:
- [ ] Kindly ensure your project has the data stats on each tab. Also, notice the header data and design kindly include it.

| Original | Yours |
| ---- | --------- |
| <img src="https://user-images.githubusercontent.com/33323077/142734206-e878057a-c9cd-44bc-8c09-389d42677d78.jpg" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/142734152-5d63f6fb-73fb-4db3-85a3-2a4579a91680.png" height="250px"/>

Reply:
Thank you for your feedback @mikenath223 .
I have implemented the changes.